src/backend/src/services/users.services.ts

```diff
@@ -214,4 +214,47 @@ export default class UsersService {
 
     return userTransformer(targetUser);
   }
+
+  /**
+   * Sets the user's secure settings
+   * @param user the user to set the secure settings for
+   * @param nuid the users nuid
+   * @param street the users street address
+   * @param city the users city
+   * @param state the users state
+   * @param zipcode the users zipcode
+   * @returns the id of the user's secure settings
+   */
+  static async setUserSecureSettings(
+    user: User,
+    nuid: string,
+    street: string,
+    city: string,
+    state: string,
+    zipcode: string,
+    phoneNumber: string
```

Comment:
we should do some validation on what we can (state, zipcode, phoneNumber). Probbably even verify it's a valid address using the google maps api

Reply:
https://developers.google.com/maps/documentation/address-validation/overview

Reply:
I thought we were gonna do this in a sperate endpoint and then send the validated stuff to this endpoint

Reply:
@Peyton-McKee sure, can you link me the ticket for that, and then I'll merge this one